EIPS/eip-2025.md

```diff
@@ -3,7 +3,7 @@ eip: 2025
 title: Block Rewards Proposal for funding Eth1.x
 author: James Hancock (@madeoftin)
 discussions-to: https://github.com/MadeofTin/EIPs/issues
-status: Draft
+status: Withdrawn
```

Comment:
There's no such status.

Reply:
Please see #2941. I suggest we merge this PR as "Abandoned" so that the PR is merged finally, and then set aside some time to collectively get this sorted once and for all.